### PR TITLE
feat(editor): add editor page styles, link to specific template

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Fix 

#### What I did

- Remove react split package
- Rename EditorCustomizer -> Editor
- Fix Styling
- Link from Home to specific template

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: src/components/pages/EditorPage.js [hunk 1]
```diff
@@ -10,14 +9,22 @@ import FolesCoverImg from '../../images/FolesCoverImg.png';
...
+import PropTypes from 'prop-types';
+import { GrDocumentText, GrClose } from 'react-icons/gr';
 
 Modal.setAppElement('#modal');
 
-export const EditorPage = () => {
+export const EditorPage = (props) => {
+  let defaultResumeIndex = 0;
+
+  if (props && props.location.hash === '#mahomes') {
```
Comment (davidholyko) [inline src/components/pages/EditorPage.js:20]: Hacky way to allow linking directly to specific template. Will refactor later

File: src/components/main/Footer.js [hunk 2]
```diff
@@ -32,7 +33,7 @@ export const Footer = () => {
           </Link>
```
Comment (PatrickDohn) [inline src/components/main/Footer.js:33]: I think we need to change this to just and a tag.

File: src/components/main/Footer.js [hunk 3]
```diff
@@ -45,7 +46,7 @@ export const Footer = () => {
         </div>
       </div>
       <div className="footer-foot">
```
Comment (PatrickDohn) [inline src/components/main/Footer.js:48]: Should this name be something other than footer-foot?

File: src/scss/index.scss [hunk 4]
```diff
@@ -20,3 +19,13 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.blank {
```
Comment (PatrickDohn) [inline src/scss/index.scss:23]: What is blank?

File: src/scss/Footer.scss [hunk 5]
```diff
@@ -7,14 +8,18 @@
...
-  justify-content: space-between;
+  justify-content: center;
+  width: 100%;
+
+  & > * {
+    margin: 0 10px;
+  }
 }
 
 .footer-content {
```
Comment (PatrickDohn) [inline src/scss/Footer.scss:25]: Do we need this if it's empty?

**Review by PatrickDohn — COMMENTED** [review]
Left some comments on a few things I found.

File: src/scss/Footer.scss (hunk 5 above)
Comment (davidholyko) [reply src/scss/Footer.scss:25]: its not empty.

https://github.com/starter-code/res-gen/blob/655868819d6622fa8e002237d74f0da8f7536876/src/scss/Footer.scss#L25-L29

File: src/components/pages/EditorPage.js [hunk 6]
```diff
@@ -75,46 +81,57 @@ export const EditorPage = () => {
...
+        >
+          Styles
+        </button>
+      </div>
+    );
+  };
+
   return (
     <div className="editor-page">
-      <Split
```
Comment (PatrickDohn) [inline src/components/pages/EditorPage.js:89]: Was this removed based on our conversation of it not doing exactly what we hoped for?

**Review by PatrickDohn — COMMENTED** [review]
I think we should change the PR title to be more in the scope of what the PR does.

File: src/components/main/Footer.js (hunk 2 above)
Comment (davidholyko) [reply src/components/main/Footer.js:33]: it looks fine to me. 

https://github.com/starter-code/res-gen/blob/655868819d6622fa8e002237d74f0da8f7536876/src/components/main/Footer.js#L31-L33


You can unlock diff separations with the blue button on bottom left. There were no no changes in these lines
Screen Shot 2020-12-11 at 12 31 16 PM

**Comment by davidholyko** [comment]
> I think we should change the PR title to be more in the scope of what the PR does.

Will do!

File: src/components/pages/EditorPage.js (hunk 1 above)
Comment (PatrickDohn) [reply src/components/pages/EditorPage.js:21]: Sounds good

**Review by PatrickDohn — APPROVED** [review]
LGTM

**Pull request merged**